source/includes/kubernetes/_k8_persistentvolumeclaims.md

```diff
@@ -285,7 +285,7 @@ Required Attributes                 | &nbsp;
 | `spec.volumemode` <br/>_string_        | What type of volume is required by the claim. Value of Filesystem is implied when not included in claim spec.|
 | `spec.dataSource` <br/>_string_        | This field can be used to specify either: an existing VolumeSnapshot object, an existing PVC or an existing custom resource/object that implements data population.|
 
-Note that the list is not complete, since it is refering to the [kubernetes api details](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#persistentvolumeclaim-v1-core).
+Note that the list is not complete, since it is refering to the [kubernetes api details](https://kubernetes.io/docs/reference/generated/kubernetes-api/api/v1.19/#persistentvolumeclaim-v1-core).
```

Comment:
This should be removed

Reply:
Fixed in commit b72186371f3f9263f21b901b1608cb3131183861